src/firefly/js/visualize/HiPSCntlr.js

```diff
@@ -1,35 +1,31 @@
-import {flux} from '../Firefly.js';
-import {get, isArray, isEmpty, set} from 'lodash';
-import {fetchUrl, parseUrl} from '../util/WebUtil.js';
+import {get, isArray, cloneDeep, set, isFunction, isUndefined, omit} from 'lodash';
```

Comment:
Is this still a `Controller` after all of the modifications?  If the answer is `no`, then let's not name it `Cntlr`.   `Controller` has a defined behavior for us.

Reply:
yes, I thought the file name should be changed. Thanks.